README.md

```diff
@@ -77,7 +94,28 @@ For ease of installation and operation, we recommend running Fleet Telemetry on
 ```
 Example: [server_config.json](./examples/server_config.json)
 
-5. Deploy and run the server. Get the latest docker image information from our [docker hub](https://hub.docker.com/r/tesla/fleet-telemetry/tags). This can be run as a binary via `./fleet-telemetry -config=/etc/fleet-telemetry/config.json` directly on a server, or as a Kubernetes deployment. Example snippet:
+### Deploy using Kubernetes with Helm Charts (recommended for large fleets)
+Hosting on [Kubernetes](https://kubernetes.io/) will enable the service to scale for large fleets. Helm Charts help you define, install, and upgrade applications on Kubernetes. Reference Helm chart [here](https://github.com/teslamotors/helm-charts/blob/main/charts/fleet-telemetry/README.md).
+
+You must have [Kubernetes](https://kubernetes.io/docs/setup/) amd [Helm](https://helm.sh/docs/intro/install/) installed for this deployment method.
+
+### Deploy using Docker
+
+1. Pull the Tesla Fleet Telemetry image (you may find all images in [docker hub](https://hub.docker.com/r/tesla/fleet-telemetry/tags)):
+```sh
+docker pull tesla/fleet-telemetry:v0.1.8
+```
+2. Run and deploy your docker image with your mTLS certificate, private key, and config.json locally mounted on `/etc/fleet-telemetry`:
```

Comment:
reword: "Run and deploy the docker image with an mTLS certificate, private key, and config.json locally mounted on `/etc/fleet-telemetry`"